src/main/java/com/transferwise/banks/demo/customer/persistence/address/AddressEntity.java

```diff
@@ -0,0 +1,52 @@
+package com.transferwise.banks.demo.customer.persistence.address;
+
+import com.transferwise.banks.demo.customer.persistence.occupation.OccupationEntity;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity(name = "customer_addresses")
```

Comment:
addresses? plural?

Reply:
I followed the naming convention. Customer is called customers so I thought I'd follow it.

Reply:
mmm I guess it's alright then